common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -45,18 +48,36 @@
                 <div class="@RenderClasses(Map("youtube-media-atom__overlay" -> true, "vjs-big-play-button" -> !expired))" style="background-image: url(@Video700.bestFor(image))">
 
                 @if(!expired) {
-                    @defining(!cardStyle.exists(c => c.toneString != "media" && !playable)) { showPlayButton: Boolean =>
-                        @if(showPlayButton) {
-                            <div class="youtube-media-atom__play-button vjs-control-text">Play Video</div>
+                    @if(immersiveMainMedia) {
+                        @defining(!cardStyle.exists(c => c.toneString != "media" && !playable)) { showPlayButton: Boolean =>
+                            @if(showPlayButton) {
+                                <div class='gs-container'>
+                                    <div class='content__main-column'>
+                                        <div class='youtube-media-atom__immersive-interface'>
+                                            <div class="youtube-media-atom__play-button vjs-control-text">
+                                                Play Video
+                                                @fragments.inlineSvg("play", "icon")
+                                            </div>
+                                            <div class="youtube-media-atom__bottom-bar__duration"></div>
+                                        </div>
+                                    </div>
+                                </div>
+                            }
                         }
-                    }
-                    @if(displayDuration) {
-                        <div class="youtube-media-atom__bottom-bar">
-                            <div class="youtube-media-atom__bottom-bar__icon">
-                            @fragments.inlineSvg("video-icon", "icon")
+                    } else {
```

Comment:
I think maybe indenation is a bit weird here? Would be good to make clear that these conditions both exist within `!expired`